src/components/views/voip/PipView.tsx

```diff
@@ -54,6 +54,11 @@ const SHOW_CALL_IN_STATES = [
     CallState.WaitLocalMedia,
 ];
 
+/**
+ * The type of a callback which will create the pip content view.
+ */
+type CreatePipContent = (options: IChildrenOptions) => JSX.Element;
```

Comment:
sorry, I didn't follow why you decided to put this here and not in `PictureInPictureDragger` ?

Reply:
Perhaps I was too keen to quick fix the strict errors here and only extracted it in the file where it is used. Moving it over and using it in more places definitively makes sense.